juan-aristizabal-bedoya/tarea11-routing/routing/src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import "./App.css";
+import { BrowserRouter, Routes, Route, Router, Link } from "react-router-dom";
+import Lunes from "./pages/Lunes";
+import Martes from "./pages/Martes";
+import Miercoles from "./pages/Miercoles";
+import Jueves from "./pages/Jueves";
+import Viernes from "./pages/Viernes";
+import Sabado from "./pages/Sabado";
+import Domingo from "./pages/Domingo";
+
+import Navbar from "./components/Navbar";
+function App() {
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Lunes />} />
```

Comment:
@Juan-Aristizabal , routes should be build dynamically from an Array using map function